src/app/core/models/product/product.helper.spec.ts

```diff
@@ -257,4 +257,153 @@ describe('Product Helper', () => {
       `);
     });
   });
+
+  describe('compare', () => {
+    let product: Product;
+    let compareProduct1: Product;
+    let compareProduct2: Product;
+
+    beforeEach(() => {
+      product = { sku: '110', inStock: true, availability: true } as Product;
+      product.attributes = [
+        {
+          name: 'Optical zoom',
+          type: 'String',
+          value: '20 x',
+        },
+        {
+          name: 'Focal length (35mm film equivalent)',
+          type: 'String',
+          value: '40 - 800 mm',
+        },
+        {
+          name: 'Image formats supported',
+          type: 'String',
+          value: '1920 x 1080, 1600 x 1200, 640 x 480',
+        },
+      ];
```

Comment:
The helper methods are only interested in the `name` property, right? The test would be more readable if the mocked data would only contain the name. This way all snapshots would be smaller.